pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -189,9 +188,19 @@ public static void main(String[] args) throws Exception {
         }
 
         if (arguments.topic != null && arguments.topic.size() != arguments.numTopics) {
-            System.out.println("The size of topics list should be equal to --num-topics");
-            jc.usage();
-            System.exit(-1);
+            // keep compatibility with the previous version
+            if (arguments.topic.size() == 1) {
+                String prefixTopicName = TopicName.get(arguments.topic.get(0)).toString();
+                List<String> defaultTopics = Lists.newArrayList();
+                for (int i = 0; i < arguments.numTopics; i++) {
+                    defaultTopics.add(String.format("%s-%d", prefixTopicName, i));
```

Comment:
This is a breaking change? The previous behavior is if the numTopics==1, the topic name should be the prefixTopicName.

Reply:
> This is a breaking change? The previous behavior is if the numTopics==1, the topic name should be the prefixTopicName.

@codelipenghui I don't think it is a breaking change since the basis of this logic is that `topic.size() != numTopics`, i.e. 
```
topics.size=1&numTopics>1: use the defaultTopics which size is numTopics and this is compatible
topics.size>1&numTopics<=1: print "The size of topics list should be equal to --num-topics" to ensure the size of them are equal
```
when  `topic.size() == numTopics`, i.e. 
```
topics.size=numTopics=1: use arguments.topic.get(0), i.e. the topic name is prefixTopicName and this is compatible
topics.size=numTopics>1: use the topic list as expected 
```

Reply:
I mean if the topics.size is 1 and topic name is `topic-a` and the numTopics=1, which topic name will be used? From your change, I think it should be `topic-a-0` right?

Reply:
No, if `topics.size=numTopics=1` the default `arguments.topic` is `[topic-a]` (which is not coverd by `defaultTopics`) and here use ` arguments.topic.get(0)` that is `topic-a`.

Reply:
Oh, I see. Thanks for the clarification.